<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="py_QWidget" name="View">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>556</width>
         <height>1125</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="generalGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="labelName">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>___NAME___</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="1" column="0">
                <widget class="QLabel" name="labelRoleTitle">
                 <property name="text">
                  <string>Role:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="labelRole">
                 <property name="text">
                  <string>___ROLE___</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QLabel" name="labelChildrenTitle">
                 <property name="text">
                  <string>Children:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="labelChildren">
                 <property name="text">
                  <string>___CHILDREN___</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="labelDescriptionTitle">
                 <property name="text">
                  <string>Description:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="labelDescription">
                 <property name="text">
                  <string>__DESCRIPTION__</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>15</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="1" column="0">
                <widget class="QLabel" name="labelPositionTitle">
                 <property name="text">
                  <string>Position:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="labelPosition">
                 <property name="text">
                  <string>___POSITION___</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="labelSizeTitle">
                 <property name="text">
                  <string>Size:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="labelSize">
                 <property name="text">
                  <string>___SIZE___</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="labelPathTitle">
                 <property name="text">
                  <string>Path:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="labelPath">
                 <property name="text">
                  <string>___PATH___</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBoxStates">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="title">
             <string>States</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QListWidget" name="listWidgetStates"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxRelations">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="title">
             <string>Relations</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QListWidget" name="listWidgetRelations">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxAttributes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Attributes</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QTreeWidget" name="treeWidgetAttributes">
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Value</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxActions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Actions</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_12"/>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxText">
          <property name="title">
           <string>Text</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="3">
            <widget class="QTextEdit" name="textEditText"/>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>124</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="buttonChangeText">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <property name="text">
              <string>Change</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/dialog-ok-apply.png</normaloff>:/icons/dialog-ok-apply.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>124</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="textVerticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxValue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Value</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="labelCurrentValue">
             <property name="text">
              <string>Current Value:</string>
             </property>
             <property name="buddy">
              <cstring>spinBoxValue</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinBoxValue">
             <property name="maximum">
              <double>999999999.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_16">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>38</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="buttonChangeValue">
             <property name="text">
              <string>Change</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/dialog-ok-apply.png</normaloff>:/icons/dialog-ok-apply.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Input</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="buttonMouse">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Mouse</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/preferences-desktop-mouse.png</normaloff>:/icons/preferences-desktop-mouse.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonKeyboard">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Keyboard</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/preferences-desktop-keyboard.png</normaloff>:/icons/preferences-desktop-keyboard.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="valueVerticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../icons/explore/icons.qrc">
     <normaloff>:/explore/icons/document-open.png</normaloff>:/explore/icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../icons/explore/icons.qrc">
     <normaloff>:/explore/icons/document-save.png</normaloff>:/explore/icons/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="icon">
    <iconset resource="../../icons/explore/icons.qrc">
     <normaloff>:/explore/icons/document-save-all.png</normaloff>:/explore/icons/document-save-all.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../icons/explore/icons.qrc">
     <normaloff>:/explore/icons/view-refresh.png</normaloff>:/explore/icons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionRefreshAll">
   <property name="icon">
    <iconset resource="../../icons/explore/icons.qrc">
     <normaloff>:/explore/icons/view-refresh.png</normaloff>:/explore/icons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh &amp;all</string>
   </property>
  </action>
  <action name="actionExpandAll">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../icons/explore/icons.qrc">
     <normaloff>:/explore/icons/go-bottom.png</normaloff>:/explore/icons/go-bottom.png</iconset>
   </property>
   <property name="text">
    <string>Expand &amp;all</string>
   </property>
  </action>
  <action name="actionExpand">
   <property name="icon">
    <iconset resource="../../icons/explore/icons.qrc">
     <normaloff>:/explore/icons/go-down.png</normaloff>:/explore/icons/go-down.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Expand</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../icons/explore/icons.qrc">
     <normaloff>:/explore/icons/tab-close.png</normaloff>:/explore/icons/tab-close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionCollapse">
   <property name="icon">
    <iconset resource="../../icons/explore/icons.qrc">
     <normaloff>:/explore/icons/go-up.png</normaloff>:/explore/icons/go-up.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Collapse</string>
   </property>
  </action>
  <action name="actionCollapseAll">
   <property name="icon">
    <iconset resource="../../icons/explore/icons.qrc">
     <normaloff>:/explore/icons/go-top.png</normaloff>:/explore/icons/go-top.png</iconset>
   </property>
   <property name="text">
    <string>Co&amp;llapse all</string>
   </property>
  </action>
  <action name="actionClearMenu">
   <property name="text">
    <string>Clear &amp;menu</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../../icons/explore/icons.qrc">
     <normaloff>:/explore/icons/edit-find.png</normaloff>:/explore/icons/edit-find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>py_QWidget</class>
   <extends>QWidget</extends>
   <header>py_qwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/explore/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
